015-quarkus-container-configuration/pom.xml

```
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.qe</groupId>
+        <artifactId>beefy-scenarios</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>015-quarkus-container-configuration</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-container-image-deployment</artifactId>
```

```suggestion
            <artifactId>quarkus-container-image</artifactId>
```

Deployment artifacts are not intended for end users

---

I know, but I'm using internal classes like ContainerImageConfig and ContainerImageProcessor.